wiki/osu!_wiki_Contribution_Guide/Common_Issues/uk.md

```diff
@@ -0,0 +1,74 @@
+# Часті проблеми при роботі з osu! wiki
+
+## Нічого не значущі описи коммітів
+
+Під цим звичайно розуміють описи коммітів з описом, начебто `Update ru.md`, які за замовчуванням створює веб-інтерфейс GitHub. Такий опис неінформативний і має малу цінність.
+
+**Добрий опис дозволяє зрозуміти суть змін, не заглядуючи всередину комміта.** Воно може бути як і коротким, так і довгим, але не більше 72 символів: залишок буде прихований в веб-інтерфейсі. Якщо ви відчуваєте, що однієї пропозиції недостатньо і при цьому ви редагуєте декілька файлів, є сенс розбити правку на декілька коммітів (по одному на кожен файл, наприклад).
```

Comment:
missing one line from the original:
```md
This can also be phrased as, "All of my commits say `Update en.md`," or similar.

If someone has told you this or you see that all of your commit messages are the same, this means you didn't use meaningful commit messages when you were able to or you tried to move, edit, and/or delete multiple files using the GitHub web interface.

A meaningful commit message lets anyone understand all of your changes within that commit without needing to see the GitHub diff. There is a 72 character limit for the commit message, so you will need to also make it short. If you have troubles trying to keep it short, you may need to split your commits. **Keep in mind that `Update en.md` or similar does not mean anything, because no one will know what specific changes you have made, nor will they know which article you had changed.**
```